types/gulp-html-prettify/gulp-html-prettify-tests.ts

```diff
@@ -0,0 +1,11 @@
+import * as gulp from 'gulp';
+import * as prettifyHtml from 'gulp-html-prettify';
+
+gulp.task('prettifyHtml', () => {
+    gulp.src('source/*.html')
+        .pipe(prettifyHtml({
+            indent_char: ' ',
```

Comment:
Maybe it's better to create the enumerate.
But, firstly, enumerate could not simply be imported from 'd.ts'.
Next, I don't know, now what else could be used. If it is the tab, how to input it?